apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -360,7 +360,10 @@ export function createApi(uri: string, networkId: NetworkID): NetworkApi {
         }
       });
 
-      return follows;
+      return follows.map(follow => {
+        follow.space.network = networkId;
```

Comment:
nit. personally I'd not mutate original object and instead spread, but not a big deal.